client/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
```

Comment:
Gitignores just means we don't show the specified files in Github. We might want to protect against our passwords, for example.